README.md

```diff
@@ -16,6 +16,22 @@ DocumentService is a library with the following functions
 - Converts an HTML document to PDF.
 - Replace placeholders in the document with actual string data.
 
+# How to setup
+
+## Steps for installing wkhtmltopdf
+- Go to website: https://wkhtmltopdf.org/downloads.html
+- Select the version of wkhtmltopdf installer that you need to download based on your system requirements.
+- Finish Installation.
+
+## Including wkhtmltopdf executable file to build package
+- Go to your class library folder that you are using to call the library.
+- Go to bin folder
+- Create a folder called "Tools" and place the wkhtmltopdf.exe file there.(wkhtmltopdf.exe can be found in the Program Files in C directory after it is installed.)
+
+We are using a Temp folder to temporarily hold the modified html file before converting it to PDF file. After the conversion is done, the temporary file is removed.
```

Comment:
Note: We are using a Temp folder to temporarily hold the modified html file before converting it to PDF file. After the conversion is done, the temporary file is removed.